### PR TITLE
Implement service unsubscribe command

### PR DESCRIPTION
**Pull request by fbiville**
Fixes issue #736

### CODE REVIEW

File: cmd/commands/service.go [hunk 1]
```diff
@@ -372,7 +377,7 @@ func ServiceSubscribe(fcClient *core.Client) *cobra.Command {
 
 	command := &cobra.Command{
 		Use:     "subscribe",
-		Short:   "Subscribe a service to an existing input channel",
+		Short:   "Subscribe a service to an existing input or output channel",
```
Comment (ericbottard) [inline cmd/commands/service.go]: Actually, not an "or", as the input is required AFAIU.

More like "and an optional output channel"

File: cmd/commands/service.go [hunk 2]
```diff
@@ -446,6 +451,34 @@ func ServiceDelete(fcClient *core.Client) *cobra.Command {
 	return command
 }
 
+func ServiceUnsubscribe(fcClient *core.Client) *cobra.Command {
+	deleteSubscriptionOptions := core.DeleteSubscriptionOptions{}
+
+	command := &cobra.Command{
+		Use:     "unsubscribe",
+		Short:   "Unsubscribe a service from an existing subscription",
+		Example: "  riff service unsubscribe my_subscription --namespace joseph-ns",
```
Comment (ericbottard) [inline cmd/commands/service.go]: Can't have underscores in k8s resource names

File: cmd/commands/service_test.go [hunk 3]
```diff
@@ -527,3 +527,86 @@ var _ = Describe("The riff service delete command", func() {
...
+	})
+
+	AfterEach(func() {
+		clientMock.AssertExpectations(GinkgoT())
+	})
+
+	It("should be documented", func() {
+		Expect(unsubscribeCommand.Name()).To(Equal("unsubscribe"))
+		Expect(unsubscribeCommand.Short).To(Equal(
+			"Unsubscribe a service from an existing subscription"))
```
[the remark this answers is not in the document]
Comment (fbiville) [reply cmd/commands/service_test.go]: I was not 100% sure. It certainly provides little value for future changes. 
I'm going to change that as you suggested.

File: cmd/commands/service.go (hunk 2 above)
Comment (fbiville) [reply cmd/commands/service.go]: Right! Let me fix that!

File: cmd/commands/service.go (hunk 1 above)
Comment (fbiville) [reply cmd/commands/service.go]: Fixing it now.

**Comment by fbiville** [comment]
Closing as a new approach has been decided (see https://github.com/projectriff/riff/issues/771 and https://github.com/projectriff/riff/issues/772).

**Pull request closed without merging**